docs/windows-design.md

```diff
@@ -0,0 +1,260 @@
+
+Table110:
+table=110, priority=200,ip,reg0=0x20000/0x20000 actions=output:gw0
+```
+
+### Use Windows named pipe for internal connections
+Named pipe is used for local connections on Windows Node instead of Unix Domain Socket (UDS). It is used in
+these scenarios:
+* OVSDB connection
+* OpenFlow connection
+* The connection between CNI plugin and CNI server
+ 
+## Antrea Management on Windows
+
+### Antrea Agent Management
+Antrea Agent is running as a process on the Windows Node, but it is managed using a DaemonSet. The utility
+[Rancher Wins](https://github.com/rancher/wins) is used to manage the host process inside the container.
+The configurations for Antrea Agent is configured using configmap, and the environment variables are set
+by kubelet on Windows.
+
+### OVS Management
+OVS is working as Windows Service. Two corresponding services are registered: ovsdb-server and ovs-vswitchd.
+
+## Traffic walk though
+
+### Intra Cluster Traffic
+
+The traffic of Pod to local Pod and Pod to remote Pod is the same as Antrea on Linux. 
+It is processed and forwarded by OVS, and controlled with OpenFlow entries.
+
+
+### Service Traffic
+
+Kube-proxy userspace mode is configured to provide the Service functions. A specific Network Adapter named as
+"HNS Internal NIC" is provided to kube-proxy to configure Service addresses. The OpenFlow entries for the Service
+traffic on Windows are the same as those on Linux.
+
+The traffic of Pod to Service is similar to Antrea on Linux. The packet from Pod to Service leaves OVS pipeline
+on the gateway interface first. kube-proxy captures the packet on the host and then chooses a valid endpoint. 
+The destination IP of the packet is changed to the endpoint, and enters the OVS pipeline again.
+
+A difference is the packet is using the client Pod's IP as source address when it enters the OVS pipeline at the
+second time on Linux, but it uses the gateway IP as source on Windows. This is because kube-proxy on Windows sets
+up a new connection on the gateway interface to the chosen backend Pod, and the packet is forwarded back to OVS
+from the new connection.
+
+### External Traffic
+
+The traffic of Pod to external address is SNATed in OVS using the Node internal IP first, and then it leaves
+the OVS pipeline on the gateway interface. As IP-Forwarding on the gateway interface is enabled, the packet
+is forwarded to the OVS bridge interface and enters the OVS for the second time. 
+When the packet enters the OVS at the second time, the dMAC is changed to an appropriate value using the host
+network stack. And OVS outputs the packet to the uplink interface directly.
+
+For the reply packet of Pod to external address, it enters the OVS from the uplink interface. It will enter
+the ct zone, and the destination IP address is recovered to the Pod IP within the ct context. Then it will be
+output to the Pod.
+
+<img src="/docs/assets/windows_external_traffic.svg" width="600" alt="Traffic to external">
+
+### Host Traffic
```

Comment:
There may be a bit of redundancy with the "OVS uplink interface configuration" section above, but that's not a big deal